### PR TITLE
Create insertion_sort.rb

### PR DESCRIPTION
**Pull request by cy-by-side**
If this pull request (PR) pertains to **Chinese-to-English translation**, please confirm that you have read the contribution guidelines and complete the checklist below:

- [ ] This PR represents the translation of a single, complete document, or contains only bug fixes.
- [ ] The translation accurately conveys the original meaning and intent of the Chinese version. If deviations exist, I have provided explanatory comments to clarify the reasons.

If this pull request (PR) is associated with **coding or code transpilation**, please attach the relevant console outputs to the PR and complete the following checklist:

- [x] I have thoroughly reviewed the code, focusing on its formatting, comments, indentation, and file headers.
- [x] I have confirmed that the code execution outputs are consistent with those produced by the reference code (Python or Java).
- [x] The code is designed to be compatible on standard operating systems, including Windows, macOS, and Ubuntu.

### CODE REVIEW

File: codes/ruby/chapter_sorting/insertion_sort.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+=begin
+File: insertion_sort.rb
+Created Time: 2024-04-02
+Author: Cy (9738314@gmail.com)
+=end
+
+### 插入排序 ###
+def insertion_sort(nums)
+  n = nums.length()
```
Comment (khoaxuantu) [inline codes/ruby/chapter_sorting/insertion_sort.rb:9]:
```suggestion
  n = nums.length
```

**Review by khoaxuantu — COMMENTED** [review]
Hi @cy-by-side, I have a small comment for this PR, the rest LGTM.

File: codes/ruby/chapter_sorting/insertion_sort.rb (hunk 1 above)
Comment (cy-by-side) [reply codes/ruby/chapter_sorting/insertion_sort.rb:9]: thx!

File: codes/ruby/chapter_sorting/insertion_sort.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+=begin
+File: insertion_sort.rb
+Created Time: 2024-04-02
+Author: Cy (3739004@gmail.com) Xuan Khoa Tu Nguyen (ngxktuzkai2000@gmail.com)
```
Comment (krahets) [inline codes/ruby/chapter_sorting/insertion_sort.rb:4]: Add a comma between the two ids.

**Review by khoaxuantu — APPROVED** [review]

File: codes/ruby/chapter_sorting/insertion_sort.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+=begin
+File: insertion_sort.rb
+Created Time: 2024-04-02
+Author: Cy (3739004@gmail.com), Xuan Khoa Tu Nguyen (ngxktuzkai2000@gmail.com)
+=end
+
+### 插入排序 ###
+def insertion_sort(nums)
+  n = nums.length
+  # 外循环：已排序区间为[0, i-1]
```
Comment (krahets) [inline codes/ruby/chapter_sorting/insertion_sort.rb:10]: `# 外循环：已排序区间为 [0, i-1]`

Please ensure the comments consistent with those in Python.

**Pull request merged**